test/crawler_web/controllers/api/v1/keyword_controller_test.exs

```diff
@@ -30,12 +30,13 @@ defmodule CrawlerWeb.Api.V1.KeywordControllerTest do
 
       keywords = Keywords.list_keywords(user.id)
 
-      list =
-        Enum.map(keywords, fn keyword ->
-          keyword.name
-        end)
+      assert length(keywords) == 3
 
-      assert list == ["one", "two", "three"]
+      created_keyword_names = Enum.map(keywords, fn k -> k.name end)
+
+      assert "one" in created_keyword_names == true
+      assert "two" in created_keyword_names == true
+      assert "three" in created_keyword_names == true
```

Comment:
Just curious, why to change it? It worked fine and it doesn't relate to current PR.

Reply:
Yeah, while writing unit testing, this is not a good test approach, so I want to include it as well, since I will not touch it until having something like chores. But, I can remove this un-related change.